docker-compose.yml

```diff
@@ -197,11 +197,8 @@ services:
     restart: always
     depends_on:
       - apigw
-      - postgres
       - auth-redis
-    depends_on:
```

Comment:
Just a side-note: version 3.0 and newer ones doesn't support the `condition`statement. As stated [here](https://stackoverflow.com/a/47714157/3753885), one solution might be delegating the verification of whether a service is up or not to its dependents.

Reply:
this is a good point that needed to be made before the version migration

Reply:
Huum. I'll check too...